src/site/xdoc/server/config-blobstore.xml

```diff
@@ -49,6 +49,34 @@
                 <dd>hybrid: Using both objectstorage for unfrequently read or big blobs & cassandra for small, often read blobs</dd>
             </dl>
 
+
+            <subsection name="Cassandra BlobStore Cache">
+                <p>A cassandra cache can be enabled to reduce latency when reading small blobs frequently.
+                    A dedicated keyspace with a replication factor of one is then used.
+                    Cached eviction policy is TTL based.
+                    Only blobs below a given threshold will be stored.
+                    To be noted that blobs are stored within a single Cassandra row, hence a low threshold should be used.
+                </p>
+                <dl>
+                    <dt><strong>cache.enable</strong></dt>
+                    <dd>DEFAULT: false, optional, must be a boolean. Whether the cache should be enabled.</dd>
+                </dl>
+                <dl>
+                    <dt><strong>cache.ttl</strong></dt>
+                    <dd>DEFAULT: 7 days, optional, must be a duration. Cached eviction policy is TTL based. </dd>
+                </dl>
+                <dl>
+                    <dt><strong>cache.timeout</strong></dt>
+                    <dd>DEFAULT: 100ms, optional, must be a duration, cannot exceed 1 hour.
+                        Timeout after which this cache should be bypassed.</dd>
+                </dl>
+                <dl>
+                    <dt><strong>cache.sizeThresholdInBytes</strong></dt>
+                    <dd>DEFAULT: 8192, optional, must be a positive integer. Unit: bytes.
```

Comment:
don't we have size units support?

Reply:
https://github.com/linagora/james-project/pull/3319/commits/00880fe3af8a5ab009556ea889a75192e35e6ab9